<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreenServerMain</class>
 <widget class="QMainWindow" name="ScreenServerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rote Augen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
      <item>
       <widget class="QLabel" name="lUserList">
        <property name="text">
         <string>Client list:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lwUserList">
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pbStart">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbInterval"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbStop">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="lServerLog">
        <property name="text">
         <string>Server log:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="lwServerLog"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
